mk/external.mk

```diff
@@ -34,14 +34,24 @@ EXTERNAL_SRC = LINUX BUSYBOX
 $(foreach T,$(EXTERNAL_SRC),$(eval $(download-n-extract)))
 
 # Build Linux kernel image
-LINUX_IMG = $(OUT)/bzImage
+ifeq ($(ARCH), x86_64)
+LINUX_IMG_NAME = bzImage
+LINUX_ARCH = x86
```

Comment:
Maybe we can modify `$(ARCH)` directly, so that we don't have to deal with `LINUX_ARCH`.

Reply:
I changed `LINUX_ARCH` to `ARCH` which means that x86 or arm64 will be assigned to `ARCH`.

However, if we directly modify `$(ARCH)`, the filename of Linux kernel configuration file should become `linux-x86.config` and `linux-arm64.config` as the variable `$(ARCH)` is used in the filename.